template/ios/RNBaseProject/Info.plist

```diff
@@ -22,6 +22,8 @@
 	<string>????</string>
 	<key>CFBundleVersion</key>
 	<string>1</string>
+	<key>CodePushDeploymentKey</key>
+	<string>CodePushDeploymentKey</string>
```

Comment:
Could you config to read it from env file

Reply:
i've updated